app/models/file_set.rb

```diff
@@ -1,5 +1,28 @@
 # Generated by hyrax:models:install
 class FileSet < ActiveFedora::Base
   include ::Hyrax::FileSetBehavior
+
+  property :rights_override, predicate: ::RDF::URI.new("http://pcdm.org/2015/06/03/rights#rightsOverride"), multiple: false do |index|
+    index.as :stored_searchable
+  end
+
+  def visibility
```

Comment:
Agree. The method `def visibility` is a duplication but yes, we can keep an eye on it for a possible future refactoring.